dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -721,12 +721,23 @@ def enter_shell(**kwargs) -> RunCommandResult:
 
     if shell_params.backend == "sqlite":
         get_console().print(
-            f"\n[warn]backend: sqlite is not "
+            f"\n[warning]backend: sqlite is not "
             f"compatible with executor: {shell_params.executor}. "
             f"Changing the executor to SequentialExecutor.\n"
         )
         shell_params.executor = "SequentialExecutor"
 
+    if (
+        shell_params.executor == "CeleryExecutor"
+        and shell_params.use_airflow_version
+        and not shell_params.airflow_extras
```

Comment:
Is it possible that the user has specified some airflow_extras but has not included celery in it? If yes, should we check that whether celery is not present and add that to the extras?

Reply:
I initially thought that if the user chooses extras, they know what they are doing - but yeah, it will be easy to add `celery` in this case if it's missing.

Reply:
Updated